ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -7,41 +7,60 @@ import UIKit
 
 /**
  `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
- Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
+ Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItemGroups` and `trailingItemGroups` add buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  */
 @objc(MSFCommandBar)
 open class CommandBar: UIView {
     // Hierarchy:
     //
-    // leadingButton
+    // leadingCommandGroupsStackView
+    // |--buttons
     // containerView
     // |--layer.mask -> containerMaskLayer (fill containerView)
     // |--subviews
     // |  |--scrollView (fill containerView)
     // |  |  |--subviews
     // |  |  |  |--stackView
     // |  |  |  |  |--buttons (fill scrollView content)
-    // trailingButton
+    // trailingCommandGroupsStackView
+    // |--buttons
 
     // MARK: - Public methods
 
-    @objc public init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
-        self.itemGroups = itemGroups
-
-        super.init(frame: .zero)
+    @available(*, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
+    @objc public convenience init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
+        var leadingItems: [CommandBarItemGroup] = []
```

Comment:
Should these two be optionals, then you wouldn't have to check for nil below and could just pass through?